third_party/xla_client/computation_client.h

```diff
@@ -220,18 +220,18 @@ class ComputationClient {
   virtual std::vector<DataPtr> TransferToServer(
       absl::Span<const TensorSource> tensors) = 0;
 
-  // Transfers local sharded tensor values to the TPU servers and returns a
-  // `PjRtShardedData`.
-  virtual DataPtr TransferShardsToServer(
-      absl::Span<const TensorSource> tensor_shards, std::string device,
-      xla::Shape shape) = 0;
-
   // Transfers local tensor values to the TPU servers and fetches the handles.
   // Update the handles associated with DataPtrs passed instead of creating new
   // datas.
   virtual void TransferToServer(absl::Span<const TensorSource> tensors,
                                 absl::Span<const DataPtr> datas) = 0;
 
+  // Transfers local sharded tensor values to the TPU servers and returns a
```

Comment:
I would use TPU Device instead of TPU Server, there is no server in PJRT context.

Reply:
Done